cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc

```diff
@@ -1,24 +1,73 @@
+See xref:prisma-cloud-compute-release-information.adoc#id79d9af81-3080-471d-9cd1-afe25c775be3[Prisma Cloud Compute Release Information] for the latest features of the host, container, and serverless capabilities that are available on the *Compute* tab on Prisma Cloud.
+//The latest release 31.00.xx is planned for August 20, 2023.
 
-//The details and functionality listed below are a preview of what is planned in the next Compute update planned for August xx, 2023; the changes listed herein and the actual release date, are subject to change.
+The details and functionality listed below are a preview of what is planned in the next Compute update planned for August 20, 2023; the changes listed herein and the actual release date, are subject to change.
 
-//=== Deprecation Notices
+=== Enhancements
+
+[cols="50%a,50%a"]
+|===
+
+|Expanded Support for Red Hat's Non-RPM Content
+|The Prisma Cloud Intelligence Stream now includes vulnerability data on non-RPM content from Red Hat, including binaries, Python scripts, JavaScript files, and Java JAR files within layered products like OpenShift.
+Rather than just flagging these as vulnerable, Prima Cloud can now leverage Red Hat's own detailed image analysis, enhancing precision in threat detection.
+
+|===
+
+=== Deprecation Notices
+[cols="50%a,50%a"]
+|===
+
+|End of Support for Cloud Native Network Segmentation (CNNS)
+|The ability to create CNNS policies that Defenders use to limit traffic from containers and hosts is being deprecated. The configuration settings on the console (*Compute > Defend > CNNS*) and the corresponding APIs for CNNS will be removed in the next major release.
+
+Radar has a container and a host view, where you can view the network topology for your containerized apps and hosts respectively, and this will continue to be available.
+
+List of API endpoints that are no longer supported:
+
+* PUT, `{{/api/v<VERSION>/policies/firewall/network/container}}`
+* GET, `{{/api/v<VERSION>/policies/firewall/network}}`
+* GET, `{{/api/v<VERSION>/audits/firewall/network/container/download}}`
+* GET, `{{/api/v<VERSION>/audits/firewall/network/container}}`
+* GET, `{{/api/v<VERSION>/audits/firewall/network/host/download}}`
+* GET, `{{/api/v<VERSION>/audits/firewall/network/host}}`
+
+|End of Support for Docker Access Control
+|Docker Access Control (`defend/access/docker`) and the Access User role (`manage/authentication/roles`)  are no longer supported.
+
+|Support for Code Repo Scanning
+|Scanning your code repositories from the Prisma Cloud Compute console at *Compute > Monitor> Vulnerabilities > Code repositories* and use of Twistcli for code repo scanning was planned for End of Support in v32.0.xxx. The deprecation is now  extended until the next major release code named O'Neal (v33.0.xxx), when the support will be dropped.
```

Comment:
End of Support is v32.00? Please confirm.

Newton is v31..00.xxx

O'Neal is v32.00.xx and not 33.0